scripts/fhap_tidy.R

```diff
@@ -0,0 +1,22 @@
+source('scripts/packages.R')
```

Comment:
https://github.com/NewGraphEnvironment/fish_passage_elk_2022_reporting/blob/main/scripts/tidy_fhap.R

Reply:
requires xref_fhap object in functions.R script from the same repo

![image](https://github.com/NewGraphEnvironment/dff-2022/assets/46538103/634a9b84-4891-4583-bc50-218bec6ba04b)